src/jit/valuenumfuncs.h

```diff
@@ -133,17 +137,29 @@ ValueNumFuncDef(JitReadyToRunNewArr, 3, false, true, false)
 ValueNumFuncDef(Box, 3, false, false, false)
 ValueNumFuncDef(BoxNullable, 3, false, false, false)
 
-ValueNumFuncDef(LT_UN, 2, false, false, false)
+ValueNumFuncDef(LT_UN, 2, false, false, false)      // unsigned or unordered comparisons
 ValueNumFuncDef(LE_UN, 2, false, false, false)
 ValueNumFuncDef(GE_UN, 2, false, false, false)
 ValueNumFuncDef(GT_UN, 2, false, false, false)
-ValueNumFuncDef(ADD_UN, 2, true, false, false)
-ValueNumFuncDef(SUB_UN, 2, false, false, false)
-ValueNumFuncDef(MUL_UN, 2, true, false, false)
 
 ValueNumFuncDef(StrCns, 2, false, true, false)
 
 ValueNumFuncDef(Unbox, 2, false, true, false)
+
+ValueNumFuncDef(ADD_UN, 2, true, false, false)      // unsigned operation
```

Comment:
Hmm, wait a moment, what is unsigned addition? The sign is only relevant for overflow checking. IL only has `add`, `add.ovf` and `add.ovf.un`.

Reply:
Let me check this

Reply:
Is it just unsigned add that doesn't exist?  I'm still thinking about subtract and multiply

Reply:
I think that only the overflow checking versions need to exist for Add, Sub and Mul